src/ucai/core/client.py

```diff
@@ -163,6 +163,20 @@ def _execute_uc_function(
         Internal logic for executing a UC function.
         """
 
+    @abstractmethod
+    def delete_function(
+        self,
+        function_name: str,
+        **kwargs,
+    ) -> None:
+        """
+        Delete a function by its full name.
+
+        Args:
+            function_name: The full name of the function to delete.
+                It should be in the format of "catalog.schema.function_name".
```

Comment:
```suggestion
                It should be in the format of "catalog.schema.function_name".
            kwargs: ...
```

to explain what kwargs is.